### DISCUSSION TITLE
Is there anyway I can execute JavaScript with any of these extensions

### DISCUSSION THREAD

**Discussion by RacoonyCoon** [3kh0/ext-remover#1404] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
![Screenshot 2024-12-19 9 59 44 AM](https://github.com/user-attachments/assets/9b2d271c-1fe5-43fd-92b9-114b2aea10c5)

**Comment by schoolexploitkid** [comment, marked as answer]
yes, read and write if u have account

**Reply by ghost** [reply]
how

**Reply by SIMONHSCH** [reply]
that's right,how?

**Reply by py660** [reply]
check the repo ur currently on

**Reply by ghost** [reply]
huh?

**Reply by py660** [reply]
go to readme.md

**Reply by ghost** [reply]
Im so confused>

**Reply by py660** [reply]
1. Locate your preferred input device (e.g. mouse, trackpad, touchscreen)
2. Swipe using one or two fingers, depending on your configuration, to cause the page to scroll up to the top
3. Move your cursor to the top-left area of the page
4. Locate the text element on the page that says "ext-remover"
5. Using your preferred input device, click on it
6. Afterwards, use your fingers to scroll down until you locate an exploit that is relevant to the extensions you have
7. A step-by-step video tutorial can be created if needed

**Reply by ghost** [reply]
yes pls

**Reply by py660** [reply]
gimme a moment

**Reply by ghost** [reply]
ok! thanks

**Reply by 88411** [reply]
Is there a video?

**Reply by py660** [reply]
@Ryler4

It has been achieved

# https://github.com/3kh0/ext-remover/discussions/1411

**Reply by RacoonyCoon** [reply]
> yes, read and write if u have account

Can you make a tutorial and teach me?
